web/src/checker/component/checker-card/checker-card.component.tsx

```diff
@@ -8,46 +23,78 @@ type Props = {
   checker: Checker
 }
 
-export const CheckerCard: FunctionComponent<Props> = ({ checker }) => (
-  <Flex
-    w="350px"
-    height="400px"
-    m="10px"
-    borderRadius="5px"
-    position="relative"
-    flexDirection="column"
-  >
-    <Heading as="h3" size="lg" textAlign="center">
-      {checker.name}
-    </Heading>
-
+export const CheckerCard: FunctionComponent<Props> = ({ checker }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  return (
     <Flex
-      border="1px solid"
-      alignItems="center"
-      justifyContent="center"
-      p="10px"
+      w="350px"
+      height="400px"
+      m="10px"
+      borderRadius="5px"
+      position="relative"
       flexDirection="column"
-      mt="15px"
     >
-      <Heading as="h2" size="md" margin="auto">
-        {checker.swapTitle}
+      <Heading as="h3" size="lg" textAlign="center">
+        {checker.name}
       </Heading>
 
-      <CheckerOracleTokenInfo oracle={checker.oracle} address={checker.address} />
+      <Flex
+        border="1px solid"
+        alignItems="center"
+        justifyContent="center"
+        p="10px"
+        flexDirection="column"
+        mt="15px"
+      >
+        <Heading as="h2" size="md" margin="auto">
+          {checker.swapTitle}
+        </Heading>
+
+        <CheckerOracleTokenInfo oracle={checker.oracle} address={checker.address} />
 
-      <HStack justifyContent="space-between" width="100%">
-        <Link to={`/checker/${checker.address}/cfmm`}>
-          <Button>Buy / Sell</Button>
-        </Link>
-        <Link to={`/checker/${checker.address}/cfmm`}>
-          <Button>Pool</Button>
-        </Link>
+        <HStack justifyContent="space-between" width="100%">
+          <Link to={`/checker/${checker.address}/cfmm`}>
+            <Button>Buy / Sell</Button>
+          </Link>
+          <Link to={`/checker/${checker.address}/cfmm`}>
+            <Button>Pool</Button>
+          </Link>
+          <>
+            <Button onClick={onOpen}>Burrows</Button>
 
-        {/* FIXME => disable is user not logged */}
-        <Link to={`/checker/${checker.address}/burrows`}>
-          <Button>Mint</Button>
-        </Link>
-      </HStack>
+            <Modal isOpen={isOpen} onClose={onClose}>
+              <ModalOverlay />
+              <ModalContent>
```

Comment:
nit - For mobile, we can add a margin to ModalContent 
 
<ModalContent m="20px"> 

![image](https://user-images.githubusercontent.com/40295238/132460187-47891174-20ed-4503-b65f-34b8409d02cb.png)

Reply:
<ModalContent m="20px">